### ISSUE TITLE
pass parameter to renderer function

### ISSUE THREAD

**Issue by makeHoulsen** [ndresx/react-countdown#170]
i need to pass a custom parameter to renderer function of each countdown,is there any solution for this?
thanks.

**Comment by ndresx** [comment]
You should be able to access any value from the outer scope within the render function, or you could use something like `partial` from, for example, lodash, to pass it in. Otherwise, there is no "direct" way to pass a custom parameter into the render function.

**Comment by makeHoulsen** [comment]
hi,
is there any way that i set id for countdown and then in rendered function get id?

**Comment by ndresx** [comment]
Yes, you can do this via lodash, or, depending on your setup, you can access it within your callback like any other variable. If you could provide an example, it might be easier to tell what can be done.
